README.md

```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+[Docker recommends that the docker engine should start after the plugins.](https://docs.docker.com/engine/extend/plugin_api/)
```

Comment:
This seems out of place here, can it be moved to a Q/A section below or an FAQ below.